src/components/Person/Person.jsx

```diff
@@ -1 +1,41 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const hasPartner = () => {
+    if (!isMarried) {
+      return 'I am not married';
+    }
+
+    return sex === 'm'
+      ? `${partnerName} is my wife`
+      : `${partnerName} is my husband`;
+  };
+
+  const renderAge = () => (
+    <p className="Person__age">
+      {`I am ${age}`}
+    </p>
+  );
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {person.age && renderAge()}
```

Comment:
I think it's redundant.

its better, when murkup placed with murkup)

and you can get age from alredy destructured person object.
Just "age" - not "person.age".

```suggestion
	{age && <p className="Person__age">{`I am ${age}`}</p>}
```